lib/games/sql/schema.sql

```diff
@@ -0,0 +1,32 @@
+
+CREATE TABLE games (
+  id int GENERATED ALWAYS AS IDENTITY(MINVALUE 0 START WITH 0 CACHE 20) NOT NULL,
+  PRIMARY KEY (id),
+  title varchar(30),
+  multiplayer boolean,
+  last_played_at date,
+  item_id int NOT NULL,
+  CONSTRAINT kf_items FOREIGN KEY(item_id) REFERENCES items(id) ON DELETE CASCADE,
+);
+
+CREATE TABLE authors (
+  id int GENERATED ALWAYS AS IDENTITY(MINVALUE 0 START WITH 0 CACHE 20) NOT NULL,
+  PRIMARY KEY (id),
+  first_name varchar(30),
+  last_name varchar(30)
+);
+
+CREATE TABLE items (
+  id int GENERATED ALWAYS AS IDENTITY(MINVALUE 0 START WITH 0 CACHE 20) NOT NULL,
+  PRIMARY KEY (id),
+  genre varchar(30),
+  author varchar(30),
+  source varchar(30),
+  label varchar(30),
+  publish_date int,
```

Comment:
I suggest changing the type to date in this line.

Reply:
Yes, that's true 👍